### PR TITLE
Button Return

### PR DESCRIPTION
**Pull request by DiegoCoronaBarragan**
# Description

We implemented a button for returning a previous page in the app if click on the company name, redirecting to the main page.

## Screenshots

image

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,10 +13,17 @@
  *= require_tree .
  *= require_self
  */
-
+.checkin_header a {
+  font-size: 30px;
+  color: white;
+  margin-right: 100px;
```
Comment (JuanequeX) [inline app/assets/stylesheets/application.css:19]: sort alphabetically this block

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -13,10 +13,17 @@
...
+}
+.checkin_header a:hover {
```
Comment (JuanequeX) [inline app/assets/stylesheets/application.css:21]: add a blank line between this two blocks

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -13,10 +13,17 @@
...
+  color: grey;
+}
```
Comment (JuanequeX) [inline app/assets/stylesheets/application.css:23]: add a blank line between this two blocks, and add a space between your class name an your curly braces

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Review by JuanequeX — APPROVED** [review]

**Review by Reiyirak — APPROVED** [review]
LGTM

**Review by Yaxkin121 — APPROVED** [review]
lgtm

**Review by MarcoRodriguez23 — APPROVED** [review]

**Pull request merged**